### PR TITLE
Fix apps settings error message timeout

### PR DESCRIPTION
**Pull request by Pytal**
See title 👆

### CODE REVIEW

File: apps/settings/src/store/apps.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const state = {
 const mutations = {
 
 	APPS_API_FAILURE(state, error) {
-		showError(t('settings', 'An error occured during the request. Unable to proceed.') + '<br>' + error.error.response.data.data.message, { timeout: 7, isHTML: true })
+		showError(t('settings', 'An error occured during the request. Unable to proceed.') + '<br>' + error.error.response.data.data.message, { timeout: TOAST_DEFAULT_TIMEOUT, isHTML: true })
```
Comment (artonge) [inline apps/settings/src/store/apps.js:42]: Isn't the timeout already set to `TOAST_DEFAULT_TIMEOUT`?

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**